java/statestore/src/main/java/sleeper/statestore/dynamodb/DynamoDBFileInfoFormat.java

```diff
@@ -108,27 +108,27 @@ Map<String, AttributeValue> createRecord(FileInfo fileInfo) throws StateStoreExc
     }
 
     FileInfo getFileInfoFromAttributeValues(Map<String, AttributeValue> item) throws IOException {
-        FileInfo fileInfo = new FileInfo();
-        fileInfo.setRowKeyTypes(rowKeyTypes);
-        fileInfo.setFileStatus(FileInfo.FileStatus.valueOf(item.get(STATUS).getS()));
-        fileInfo.setPartitionId(item.get(PARTITION).getS());
+        FileInfo.Builder fileInfo = FileInfo.builder();
+        fileInfo.rowKeyTypes(rowKeyTypes);
+        fileInfo.fileStatus(FileInfo.FileStatus.valueOf(item.get(STATUS).getS()));
+        fileInfo.partitionId(item.get(PARTITION).getS());
+        fileInfo.filename(item.get(NAME).getS());
```

Comment:
Think the lines above should use the builder, i.e. 

    FileInfo.Builder fileInfo = FileInfo.builder()
         .rowKeyTypes(rowKeyTypes)
         .fileStatus(FileInfo.FileStatus.valueOf(item.get(STATUS).getS())
         .partitionId(item.get(PARTITION).getS())
         .filename(item.get(NAME).getS());

Reply:
Fixed in cac33ce